pkg/console/operator/sync_v400.go

```diff
@@ -301,6 +301,8 @@ func SyncConfigMap(co *consoleOperator, recorder events.Recorder, operatorConfig
 		return nil, false, mcErr
 	}
 
+	copyCustomLogoConfigMap(co, operatorConfig)
```

Comment:
With this approach, I don't know that I would bury it down inside `SyncConfigMap()`.  Any reason to not make the call a sibling to `SyncConfigMap()` and `SyncServiceCAConfigMap()` a level up?